source/_components/proximity.markdown

```diff
@@ -47,6 +47,7 @@ proximity:
     - device_tracker.eleanorsiphone
     - device_tracker.tsiphone
   tolerance: 50
+  unit_of_measurement: mi
```

Comment:
It allows you to have the proximity in a different unit than the
application wide unit system. It should be marked optional though

On Aug 7, 2016 6:59 PM, "Paulus Schoutsen" <notifications@github.com> wrote:

> In source/_components/proximity.markdown
> <https://github.com/home-assistant/home-assistant.io/pull/698#discussion_r73813102>
> :
>
> > @@ -47,6 +47,7 @@ proximity:
> >      - device_tracker.eleanorsiphone
> >      - device_tracker.tsiphone
> >    tolerance: 50
> > +  unit_of_measurement: mi
>
> Is this one still relevant?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant.io/pull/698/files/1c474fca52a0c0fce5e2fb8878861b14b44eaa2d#r73813102>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZaqo70UfmcNePJ5AWkrsp292lE4vks5qdn-DgaJpZM4JYthw>
> .
>